setup.py

```diff
@@ -40,25 +40,31 @@
       test_suite='nose.collector',
       packages=find_packages(),
       include_package_data=True,
-      package_data={'labcontrol.db': [
-        'support_files/skeleton_labcontrol.cfg',
-        'support_files/labcontrol.dbs',
-        'support_files/labcontrol.html', 'support_files/db_patch.sql'],
-        'labcontrol.gui': ['templates/*.html',
-                           'static/js/*.js',
-                           'static/img/*.gif',
-                           'static/img/*.ico',
-                           'static/img/*.png',
-                           'static/css/*.css',
-                           'static/css/images/*.gif',
-                           'static/vendor/js/*.js',
-                           'static/vendor/js/slickgrid.plugins/*.js',
-                           'static/vendor/css/*.css',
-                           'static/vendor/css/*.map',
-                           'static/vendor/css/images/*.gif',
-                           'static/vendor/css/images/*.png',
-                           'static/vendor/fonts/glyphicons*.*',
-                           'static/vendor/licenses/*_LICENSE']},
+      package_data={
+        'labcontrol.db': [
+            'support_files/*',
+            'support_files/patches/*'],
```

Comment:
As far as I know, this is standard practice. I do agree with you, however. What's important however, is that all python code passes flake8. Not all code may be going through flake8 during CI testing, but this should be a separate issue.

Yes, this change installs most of the source base during a 'pip install'. This change was made because too many necessary files were not being installed and the Travis build was failing. At some point it will be good to prune what isn't necessary, but that is a separate issue.